docs/installation.md

```diff
@@ -0,0 +1,163 @@
+- Docker registry url (REGISTRY_URL)
+
+
+## Installation steps
+
+### 1.Install tiller on kubernetes cluster
+
+```
+kubectl -n kube-system create serviceaccount tiller
+
+kubectl create clusterrolebinding tiller \
+  --clusterrole cluster-admin \
+  --serviceaccount=kube-system:tiller
+
+helm init --service-account tiller
+```
+
+### 2.Clone inference-model-manager repo
+```
+git clone git@github.com:IntelAI/inference-model-manager.git
+```
+### 3.Build CRD controller image and push to docker registry
+```
+cd inference-model-manager/server_controller
+make docker_build
+```
+obtain IMAGE_ID using 
+```
+docker images
+```
+tag and push the image
+```
+docker tag $IMAGE_ID $REGISTRY_URL/server-controller-prod:latest
+docker push $REGISTRY_URL/server-controller-prod:latest
+```
+### 4.Configure CRD chart and install it 
+```
+vim inference-model-manager/helm-deployment/crd-subchart/values.yaml
+- replace 
+<crd_image_path> with $REGISTRY_URL/server-controller-prod
+<crd_image_tag> with latest 
+<dns_domain_name> with imm.example.com
+cd inference-model-manager/helm-deployment/crd-subchart/
+helm install .
+```
+Check installation
+Issue command
+``` 
+kubectl get pods -n crd
+```
+Expected output:
+```
+NAME                                READY     STATUS             RESTARTS   AGE
+server-controller-c989895b7-pvh55   1/1       Running   0                   17s
+```
+
+### 5. Install DEX Oauth2Server [dex doc]
+In this step you need to configure DEX connection to identity provider, like LDAP.
+Sample dex configuration for LDAP: https://github.com/dexidp/dex/blob/master/examples/config-ldap.yaml
+```
+cd inference-model-manager/helm-deployment/dex-subchart
+vim values.yaml
+# enter dex configuration here, use link above if necessary
+helm install .
+```
+After this step dex pod should be running in dex namespace:
+
+Execute:
+```
+kubectl get pods -n dex
+```
+Expected output:
+```
+
+NAME                   READY     STATUS    RESTARTS   AGE
+dex-6f8d94bd5f-9vlvm   1/1       Running   1          1m
+```
+### 6. Build Management API and push image to registry
+```
+cd inference-model-manager/management
+make docker_build
+```
+get the IMAGE_ID using 
+```
+docker images
+```
+Tag and push the image to registry
+```
+docker tag $IMAGE_ID $REGISTRY_URL/management-api:latest
+docker push $REGISTRY_URL/management-api:latest
+```
+
+### 7. Install Management API [management api doc]
+In this step it's important to setup following variables in values.yaml file:
+
+- <image_path> $REGISTRY_URL/management-api
+- <image tag> use "latest"
+- <management_api_desired_dns> should be a subdomain of your FQDN, like mgt.imm.example.com
+- <dns_for_inference_endpoints> could be your domain name, like imm.example.com
+- <minio_access_key>  Minio compatible storage access key
+- <minio_secret_key> Minio compatible storage secret key
+- <minio_endpoint> not used, set to s3.<region>.amazonaws.com
+- <minio_endpoint_url> s3.<region>.amazonaws.com
```

Comment:
< region > is cutted if you look [here](https://github.com/IntelAI/inference-model-manager/blob/update-docs/docs/installation.md) so it looks like two dots next to each other